index.js

```diff
@@ -0,0 +1,9 @@
+const express = require('express')
+const villains = require('./villains')
+const {getVillains} = require('./controller/villains.js')
+const app = express()
+app.get('/', getVillains)
+app.all('*',)
+app.listen(2020, () => {
+    console.log('hi')
```

Comment:
your console log for your server should either be informative (which port is being listened on) or nothing at all.